README.md

```diff
@@ -1,15 +1,28 @@
-# Calendar-Proxy
+# TUM Calendar Proxy
 
+This is a proxy service that simplifies and enhances the iCal export from TUM Online. It allows you to:
```

Comment:
Do you think adding a picture like on <https://cal.tum.app> would be worth it?

Reply:
yes for sure

Reply:
Interesting, that the "Marketing website" is not served from this repo.. 🤷🏻‍♂️